docs/proxy/google_serp_proxy/nodejs_examples.md

```diff
@@ -17,7 +17,11 @@ Get a list of results from the US for keyword `wikipedia` and parse them through
 
     Apify.main(async() => {
         const password = process.env.APIFY_PROXY_PASSWORD;
-        const proxyUrl = `http://groups-GOOGLE_SERP:${password}@localhost:8000`;
+        const proxyConfiguration = await Apify.createProxyConfiguration({
+            password,
+            groups: ['GOOGLE_SERP'],
+            countryCode: 'US',
```

Comment:
I don't think `GOOGLE_SERP` supports `countryCode`, but I might be wrong.

Reply:
You are right according to the docs. Changed.